rcsb/db/config/exdb-config-example.yml

```diff
@@ -119,21 +122,26 @@ site_info_configuration:
   PDBX_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/development/dictionary_files/reference/mmcif_pdbx_v5_next.dic
   RCSB_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/development/dictionary_files/dist/rcsb_mmcif_ext.dic
   RCSB_COMP_MODEL_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/development/dictionary_files/dist/rcsb_mmcif_comp_model_ext.dic
-  IHMDEV_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/development/dictionary_files/reference/ihm-extension.dic
   FLR_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/development/dictionary_files/reference/flr-extension.dic
   VRPT_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/development/dictionary_files/reference/vrpt_mmcif_ext.dic
   VRPT_DICT_MAPPING_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/development/dictionary_files/reference/vrpt_dictmap.json
-  #
+  IHMDEV_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/development/dictionary_files/reference/ihm-extension.dic
   MA_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/development/dictionary_files/reference/mmcif_ma.dic
+  ## Previously pointed to 'dictionary_files/reference', but point to authoritative source instead to ensure most recent version is used
```

Comment:
I've only done so a few times, but haven't pulled the latest in a while so that we know we have a stable version for development purposes. However, I added that comment in the config file because I think in the future, we'd want to point to the authoritative source directly; but I can also see the drawback to that approach. So, I figured this would be something we should make a decision on later.

Reply:
There may be breaking changes if we pull from the authoritative source. It is perhaps better to pull routinely knowing what changes have been made.

Reply:
OK, that makes sense. Sounds good with me, thanks.